docs/src/appendix/snforge.md

```diff
@@ -0,0 +1,8 @@
+# Forge CLI Reference
+
+* [`snforge test`](./snforge/test.md)
+* [`snforge init`](./snforge/init.md)
+* [`snforge clean-cache`](./snforge/clean-cache.md)
+
+You can check your version of Forge via `snforge --version`.
```

Comment:
```suggestion
You can check your version of snforge via `snforge --version`.
```
? I'm not sure

Reply:
the same goes for multiple places here (like appendix/snforge-library/print.md etc), how should we approach it? I guess it should also be changed? (there aren't any occurences of cast | Cast, that's why I wonder)

(to see where forge | Forge is still used I used: )
```
starknet-foundry/docs/src $ ➜ grep -rnw -e "forge"
    and
starknet-foundry/docs/src $ ➜ grep -rnw -e "Forge"
```

Reply:
thanks for the tip @THenry14 <3